packages/boot/test/bootstrapTests/liquidation.ts

```diff
@@ -217,6 +219,39 @@ export const makeLiquidationTestContext = async t => {
   };
 };
 
+export const addSTARsCollateral = async t => {
```

Comment:
please add the type
```suggestion
export const addSTARsCollateral = async (
  t: ExecutionContext<LiquidationTestContext>,
) => {
```
You'll have to move up `export type LiquidationTestContext` (which should be run under `makeLiquidationTestContext`) regardless).

Reply:
I see a thumbs-up but not the change